tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/webapp/security/sso/AuthenticationIT.java

```diff
@@ -458,15 +389,6 @@ public void testLoginAndUsageOfRestApi(final BiFunction<String, String, Tokens>
     assertThat(currentUserAfterLogin.getSalesPlanType())
         .isEqualTo(TASKLIST_TEST_SALESPLAN.getType());
     assertThat(currentUserAfterLogin.getRoles()).isEqualTo(TASKLIST_TEST_ROLES);
-    assertThat(currentUserAfterLogin.getC8Links())
-        .extracting("name", "link")
-        .containsExactly(
-            tuple("console", "https://console.audience/org/3/cluster/test-clusterId"),
-            tuple("operate", "http://operate-url"),
-            tuple("optimize", "http://optimize-url"),
-            tuple("modeler", "https://modeler.audience/org/3"),
-            tuple("tasklist", "http://tasklist-url"),
-            tuple("zeebe", "grpc://zeebe-url"));
```

Comment:
Why was this assertion removed?

Reply:
We are not using this anymore, since we are now using the C3 component, the links are not needed.